CGAL_Core/include/CGAL/CORE/BigInt.h

```diff
@@ -24,86 +24,80 @@
 #ifndef _CORE_BIGINT_H_
 #define _CORE_BIGINT_H_
 
+#include <CGAL/boost_mp.h>
 #include <CGAL/CORE/Gmp.h>
 #include <CGAL/CORE/RefCount.h>
 #include <CGAL/CORE/MemoryPool.h>
 #include <string>
 
 namespace CORE {
 
+#ifdef CGAL_CORE_USE_GMP_BACKEND
```

Comment:
Similar question here. We recommend to use at least boost 1.79 for boost backend. Should we take that into account for defining `CGAL_CORE_USE_GMP_BACKEND`?